module/VuFind/src/VuFind/Config/YamlReader.php

```diff
@@ -179,6 +182,22 @@ protected function parseYaml($file, $defaultParent = null)
             // Swallow the directive after processing it:
             unset($results['@parent_yaml']);
         }
+        // Override default parent with explicitly-defined parent, if present:
```

Comment:
This comment is identical to the previous code block; we should probably differentiate:

```suggestion
        // Override default parent with a named configuration, if present:
```